lua/core/screen.lua

```diff
@@ -477,4 +557,22 @@ Screen.blend_mode = function(index)
   end
 end
 
+--@tparam number/boolean i
+--
+-- if number:
+-- i == 0 -> normal mode
+-- i < 0 -> inverted mode
+-- i > 0 -> inverted mode
+--
+-- if boolean:
+-- false -> normal mode
+-- true -> inverted mode
+Screen.invert = function(i)
+  if type(i) == "boolean" then
+    _norns.screen_invert(1 and i or 0)
```

Comment:
oo should this be `_norns.screen_invert(i and 1 or 0)`?

Reply:
> oo @tlubke should this be _norns.screen_invert(i and 1 or 0)?

yes 😔

Reply:
fixed